src/content/guidelines/content/guidance.md

```diff
@@ -1,9 +1,18 @@
 ---
-label: Guidelines
+label: Using the appropriate voice, tone and terminology allows us to better connect and resonate with our users. 
 title: Content
 tabs: ['General', 'Guidance', 'Glossary']
 ---
 
+
+<anchor-links>
+<ul>
+    <li><a href="#capitalization">Capitalization</a></li>
+    <li><a href="#do’s--dont’s">Do’s & dont’s</a></li>
```

Comment:
this link won't work with special characters.  I'll open a separate dev issue to fix so that the charcter will be ignored. 
```suggestion
    <li><a href="#dos--donts">Do’s & dont’s</a></li>
```

Reply:
Ok thanks, @aagonzales and I were trying to figure that out!